dbm.opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "1.2"
+maintainer: "https://github.com/ocaml/opam-repository/issues"
+authors: "Francois Rouaix et. al."
+license: "LGPL-v2 with OCaml linking exception"
+homepage: "https://github.com/ocaml/dbm"
+bug-reports: "https://github.com/ocaml/dbm/issues"
+dev-repo: "https://github.com/ocaml/dbm.git"
+build: [
+  ["./configure"]
+  ["jbuilder" "build" "-p" name "-j" jobs]
+  ["jbuilder" "build" "testdbm.exe"]
```

Comment:
I think this should probably me more in a separate `build-test` field

Reply:
Also the stardard way of doing tests with jbuilder seems to be `jbuilder runtest -p name`

Reply:
ok, thanks for educating me.
By the way, funnily, jbuilder runtest -p dbm, will only run tests after a fresh build. I.e. only once.
This might be a jbuilder bug.

Reply:
By the way, I do I tell opam to run those tests now?
Apparently they are not run by a regular install of the package.